functions/go/format/README.md

```diff
@@ -0,0 +1,73 @@
+# format
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Format resources in a directory.
```

Comment:
"in a directory" is both inaccurate (it should format all resources in a package) and out of scope (scope of a function is not up to the function but defined by the orchestrator).

Reply:
Updated.